### ISSUE TITLE
CapX capacity timeline empty space inconsistent

### ISSUE THREAD

**Issue by ianhinder** [UoMResearchIT/CapX#606]
In the CapX capacity view for an individual RSE, there are sometimes gaps shown in white in the "total" row. These appear to have no bar, and don't show any availability information. I initially interpreted this as the RSE being unavailable, e.g. due to planned time off. But when I went back to the capacity page for all RSEs, it was shown as green and as "fully available". 

Should the gap in the individual page in the total row really be a green bar, to be consistent?

Adrian (in PSM chat):
> Possibly 😀

**Comment by Scottan** [comment]
Just a note that I spotted this today as well when looking at Benito's availability and it caused me some confusion (see below):

![image](https://github.com/user-attachments/assets/bcb40611-746c-4c22-b1e6-f40151d92539)

I raised a ticket before @ianhinder found that it was a duplicate of this one

**Comment by aharwood2** [comment]
I don't think this exists as a problem anymore. It looks to have been fixed along the way.
